src/services/todos.js

```diff
@@ -0,0 +1,21 @@
+import { client, checkError } from './client';
+
+export async function getTodos() {
+  const resp = await client.from('todos_react').select('*');
+  return checkError(resp);
+}
```

Comment:
We can add an orderBy method on the end so that the list doesn't change when a task is completed.